distributions/Dist_vag273_2.py

```diff
@@ -0,0 +1,22 @@
+
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_vag273(BaseDistribution):
```

Comment:
this class needs to be named Dist_vag273_2. If you make a change and sync to your GitHub account, the pull request will be automatically updated